### PR TITLE
new icon: postman (original, original-wordmark)

### LINKED ISSUE #933: Icon request: Postman
### About the icon

Post is an api tool widely used by developers

### Links and sources

https://www.vectorlogo.zone/util/preview.html?image=/logos/getpostman/getpostman-icon.svg

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by kilian-paquier**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes #933 #1278 #159**

## Link to prove your SVG is correct and up-to-date.

https://www.postman.com/

### CODE REVIEW

File: devicon.json [hunk 1]
```diff
@@ -7195,5 +7195,25 @@
...
+            "svg": [
+                "original",
+                "original-wordmark"
+            ],
+            "font": [
+                "plain",
+                "plain-wordmark"
+            ]
+        },
+        "color": "#ff6c37",
```
Comment (Snailedlt) [inline devicon.json:7216]: This color should be `#f37036`
image

File: icons/postman/postman-plain-wordmark.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
... .5-.156 1.192.535 1.192.645 0 .493-.697.493-1.14v-8.61c0-.448.136-.989-.395-1.14-.543-.099-.793.347-.988.792l-1.52 3.266c-.152.297-.2.89-.644.89-.4-.062-.446-.593-.594-.89l-1.524-3.364c-.17-.389-.42-.744-.812-.724zm-65.664.113a.458.458 0 0 0-.09.082l.09-.082zm21.133.754c.476.006.933.12 1.328.516.746.789.742 1.78.496 2.818-.247 1.043-.789 1.389-2.227 1.389-1.386 0-1.386 0-1.386-1.586v-2.227c0-.348 0-.79.347-.79.471-.027.966-.126 1.442-.12zm11.824.068c1.138-.05 1.829.697 1.879 1.98v2.54c0 .891.05 1.742 0 2.574-.047 1.34-.741 2.028-1.88 2.028-1.104-.051-1.746-.747-1.796-2.028v-5.113c.066-1.29.712-1.98 1.797-1.98zm-32.057.52c-.484 1.785-2.84 4.182-5.326 6.535l-1-1 6.326-5.535zm80.541.621c.297 0 .247.344.297.594l.942 3.512c.343 1.234.293 1.285-.793 1.285h-.496l.003.02c-1.585 0-1.636 0-1.14-1.798l.793-2.968c.05-.301.097-.645.394-.645zm-89.86 2.68.028.025.985.985-2.575.552 1.563-1.562zm1.548 1.547.328.336a.302.302 0 0 0 .004.021l.017.002.3.305-1.534.232-.004-.008.889-.888zm1.012 1.103.98.97a34.022 34.022 0 0 1-6.91 4.45.387.387 0 0 0-.207.441l.277 1.204a.071.071 0 0 1 0 .093.083.083 0 0 1-.11 0l-1.628-1.656 5.045-5.117 2.553-.385zm-7.965 5.861.117.11 1.307 1.306-2.674-.173 1.25-1.243z"/>
```
Comment (Snailedlt) [inline icons/postman/postman-plain-wordmark.svg:2]: Change path fill to correct color -> `#f37036`
```suggestion
    <path fill="#f37036" d="M21.95 47.426a16.584 16.584 0 0 0-4.833.57A16.577 16.577 0 0 0 5.02 61.891a16.579 16.579 0 0 0 8.199 16.496 16.58 16.58 0 0 0 18.379-1.272c7.253-5.596 8.565-16.002 2.984-23.248a16.584 16.584 0 0 0-12.633-6.441zm6.952 6.18a2.48 2.48 0 0 1 1.493.511l-2.153 2.168a.209.209 0 0 0-.076.123v.004a.233.233 0 0 0-.006.045.217.217 0 0 0 0 .002v.002a.227.227 0 0 0 .006.027.217.217 0 0 0 .076.145l1.63 1.629a2.414 2.414 0 0 1-2.536-.38l-.008-.005c-.004-.004-.007-.01-.012-.014a.257.257 0 0 0-.007-.004 2.45 2.45 0 0 1-.118-.109 2.426 2.426 0 0 1-.52-2.64 2.424 2.424 0 0 1 2.231-1.505zm1.903.917c.35.425.543.96.543 1.516a2.43 2.43 0 0 1-.72 1.727 2.46 2.46 0 0 1-.278.23l-1.584-1.543 2.039-1.93zm-.159 1.106a.207.207 0 0 0-.084.02.247.247 0 0 0-.105.113c-.027.05-.012.105.012.152a.59.59 0 0 1-.07.621.184.184 0 0 0 0 .262.143.143 0 0 0 .113 0 .183.183 0 0 0 .18-.055.956.956 0 0 0 .124-1.008l.008-.004a.207.207 0 0 0-.178-.101zm-5.234 1.826a1.494 1.494 0 0 1 1.045.326 1.266 1.266 0 0 0-.73.356l-6.092 6.05L18.445 63c3.157-3.14 5.087-4.767 6.285-5.32a1.494 1.494 0 0 1 .682-.225zm1.082.768h.002a.769.769 0 0 1 .668.218.733.733 0 0 1 .065.069 1.567 1.567 0 0 1 .12.21.733.733 0 0 1 .006.014.733.733 0 0 1 .043.27.77.77 0 0 1-.261.555L20.64 65.21l-.694-.691 6.121-6.08a.769.769 0 0 1 .426-.217zm45.041.166a2.644 2.644 0 0 0-.547.05c-1.735.347-2.527 1.325-2.379 3.118.098 1.34.844 1.978 1.832 2.423.692.36 1.395.596 1.93 1.19a2.125 2.125 0 0 1 .445 2.226c-.3.843-.843 1.188-1.582 1.239-.793.039-1.438-.144-2.03-.79-.213-.25-.493-.45-.743-.154-.297.348-.047.694.148.944.65.788 1.484.986 2.524.986v-.012c.347-.1.89-.048 1.387-.396 1.089-.641 1.386-1.633 1.386-2.918 0-1.387-.593-2.336-1.683-2.871-.692-.344-1.434-.555-2.078-1.086-.496-.45-.595-1.04-.497-1.734.101-.692.606-1.04 1.141-1.137.645-.098 1.338-.098 1.98.25.299.144.64.343.79-.154.148-.43-.149-.727-.493-.829-.483-.184-.995-.34-1.53-.345zm36.342.02c-.527.032-.577.626-.717 1.04l-2.426 8.961c-.097.395-.246.891.25 1.086.493.137.543-.344.645-.691.593-1.798.593-1.797 2.273-1.797h.438c1.352 0 1.353 0 1.793 1.52.052.201.149.449.199.644.09.297.285.347.531.297.297-.051.348-.297.348-.645l-.008.028a5.705 5.705 0 0 0-.195-.891l-2.278-8.512c-.097-.442-.207-.989-.742-1.039a.81.81 0 0 0-.111-.002zm-48.373.032c-1.484 0-2.472.691-2.672 2.13-.297 2.22-.246 4.49 0 6.718.199 1.538 1.141 2.23 2.773 2.23 1.536 0 2.375-.757 2.622-2.277.15-1.087.052-2.124.101-3.207-.05-1.14.098-2.278-.152-3.465-.246-1.387-1.188-2.129-2.672-2.129zm20.441.004c-.245 0-.445.2-.445.496 0 .383.203.538.496.538s.642.046.938 0c.743-.102 1.089.197 1.039 1.134v7.969c0 .383 0 .828.496.828.446 0 .496-.433.496-.828v-8.115c0-.742.2-1.09.89-1.04.348.05.743 0 1.086 0 .301 0 .496-.187.496-.534 0-.293-.195-.448-.445-.448h-5.047zm-34.207.008c-.582 0-.828.246-.828.988v4.586l-.023.018v4.65c0 .395 0 .793.445.793s.492-.394.492-.742c0-.828.051-1.733 0-2.623-.047-.743.203-1.088.895-1.041.453.05.836 0 1.281 0 1.547-.098 2.438-1.12 2.488-2.918.051-2.625-.793-3.66-3.07-3.71h-1.68zm76.961.106c-.285 0-.481.152-.531.496-.051.246-.05.441-.05.691v5.8c0 .2.05.497-.196.599-.301.097-.3-.199-.399-.399-.148-.398-.347-.742-.496-1.137l-2.422-5.398c-.203-.395-.45-.744-.89-.594-.398.1-.348.548-.348.89v9.122c0 .394 0 .939.535.89.496 0 .395-.546.395-.89v-5.89c0-.35-.099-.743.2-1.04l.01-.02c.337.147.388.493.536.79l2.77 6.136c.2.446.407.988.94.793.494-.153.345-.69.345-1.09v-9.156c0-.297-.102-.543-.399-.593zm-30.531.029a.722.722 0 0 0-.176.033c-.445.15-.344.691-.344 1.086v9.305c.047.246.149.445.446.496.285 0 .481-.2.531-.496.05-.297.05-.596.05-.89v-5.99c0-.185-.05-.484.2-.53.195-.051.195.246.297.394l1.332 2.922c.152.344.199.738.598.738.28 0 .383-.394.53-.691l1.634-3.512.007-.016c.395.4.294.793.344 1.188v5.691c0 .5-.156 1.192.535 1.192.645 0 .493-.697.493-1.14v-8.61c0-.448.136-.989-.395-1.14-.543-.099-.793.347-.988.792l-1.52 3.266c-.152.297-.2.89-.644.89-.4-.062-.446-.593-.594-.89l-1.524-3.364c-.17-.389-.42-.744-.812-.724zm-65.664.113a.458.458 0 0 0-.09.082l.09-.082zm21.133.754c.476.006.933.12 1.328.516.746.789.742 1.78.496 2.818-.247 1.043-.789 1.389-2.227 1.389-1.386 0-1.386 0-1.386-1.586v-2.227c0-.348 0-.79.347-.79.471-.027.966-.126 1.442-.12zm11.824.068c1.138-.05 1.829.697 1.879 1.98v2.54c0 .891.05 1.742 0 2.574-.047 1.34-.741 2.028-1.88 2.028-1.104-.051-1.746-.747-1.796-2.028v-5.113c.066-1.29.712-1.98 1.797-1.98zm-32.057.52c-.484 1.785-2.84 4.182-5.326 6.535l-1-1 6.326-5.535zm80.541.621c.297 0 .247.344.297.594l.942 3.512c.343 1.234.293 1.285-.793 1.285h-.496l.003.02c-1.585 0-1.636 0-1.14-1.798l.793-2.968c.05-.301.097-.645.394-.645zm-89.86 2.68.028.025.985.985-2.575.552 1.563-1.562zm1.548 1.547.328.336a.302.302 0 0 0 .004.021l.017.002.3.305-1.534.232-.004-.008.889-.888zm1.012 1.103.98.97a34.022 34.022 0 0 1-6.91 4.45.387.387 0 0 0-.207.441l.277 1.204a.071.071 0 0 1 0 .093.083.083 0 0 1-.11 0l-1.628-1.656 5.045-5.117 2.553-.385zm-7.965 5.861.117.11 1.307 1.306-2.674-.173 1.25-1.243z"/>
```

File: icons/postman/postman-plain.svg [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
....625zm7.12 3.432a8.87 8.87 0 0 1 2.033 5.674 9.15 9.15 0 0 1-2.688 6.464 9.989 9.989 0 0 1-1.098.895L92.307 36.7l-.963-.963.265-.265 7.373-6.96zm-.366 4.193a.777.777 0 0 0-.55.031.731.731 0 0 0-.36.426.73.73 0 0 0 .05.559 2.226 2.226 0 0 1-.257 2.328.64.64 0 0 0-.195.488c.004.184.07.36.195.492a.58.58 0 0 0 .414 0 .68.68 0 0 0 .672-.207 3.573 3.573 0 0 0 .465-3.777v.004a.777.777 0 0 0-.434-.344zM79.34 39.43a5.584 5.584 0 0 1 3.31 1.226 4.756 4.756 0 0 0-2.681 1.34L57.162 64.701l-4.476-4.476c11.828-11.772 19.06-17.921 23.556-19.936a5.584 5.584 0 0 1 3.098-.86zm3.965 2.96a2.895 2.895 0 0 1 2.043.844 2.786 2.786 0 0 1 .879 2.121 2.869 2.869 0 0 1-.985 2.07l-24.25 21.106-2.617-2.617 22.887-22.68a2.895 2.895 0 0 1 2.043-.843zm2.994 6.698c-1.69 6.702-10.647 15.783-19.987 24.607l-3.777-3.773L86.3 49.088zM51.367 61.547l.274.27 3.513 3.513-9.63 2.06 5.843-5.843zm5.793 5.84.004.004 1.168 1.195a1.086 1.086 0 0 0 .018.084l.078.012.248.254.82.84-5.385.66 3.05-3.05zm3.867 4.076 3.578 3.576A126.992 126.992 0 0 1 38.75 91.695a1.44 1.44 0 0 0-.777 1.653l1.035 4.5a.31.31 0 0 1 0 .363.31.31 0 0 1-.414 0l-6.102-6.152L51.3 72.975l9.728-1.512zm-29.933 21.94.869.814 4.492 4.492-10.016-.648 4.655-4.659z"/>
```
Comment (Snailedlt) [inline icons/postman/postman-plain.svg:2]: Change path fill to correct color -> `#f37036`

File: icons/postman/postman-original.svg [hunk 4]
```diff
@@ -0,0 +1,5 @@
+<svg viewBox="0 0 128 128" xmlns="http://www.w3.org/2000/svg">
+    <path fill="#ff6c37" d="M113.117 26.066C92.168-1.062 53.191-6.07 26.062 14.883c-27.125 20.953-32.128 59.93-11.175 87.055 20.957 27.124 59.937 32.124 87.058 11.167 27.114-20.953 32.118-59.918 11.172-87.039Zm0 0"/>
```
Comment (Snailedlt) [inline icons/postman/postman-original.svg:2]:
```suggestion
    <path fill="#f37036" d="M113.117 26.066C92.168-1.062 53.191-6.07 26.062 14.883c-27.125 20.953-32.128 59.93-11.175 87.055 20.957 27.124 59.937 32.124 87.058 11.167 27.114-20.953 32.118-59.918 11.172-87.039Zm0 0"/>
```

File: icons/postman/postman-original.svg [hunk 5]
```diff
@@ -0,0 +1,5 @@
...
....625zm7.12 3.432a8.87 8.87 0 0 1 2.033 5.674 9.15 9.15 0 0 1-2.688 6.464 9.989 9.989 0 0 1-1.098.895L92.307 36.7l-.963-.963.265-.265 7.373-6.96zm-.366 4.193a.777.777 0 0 0-.55.031.731.731 0 0 0-.36.426.73.73 0 0 0 .05.559 2.226 2.226 0 0 1-.257 2.328.64.64 0 0 0-.195.488c.004.184.07.36.195.492a.58.58 0 0 0 .414 0 .68.68 0 0 0 .672-.207 3.573 3.573 0 0 0 .465-3.777v.004a.777.777 0 0 0-.434-.344zM79.34 39.43a5.584 5.584 0 0 1 3.31 1.226 4.756 4.756 0 0 0-2.681 1.34L57.162 64.701l-4.476-4.476c11.828-11.772 19.06-17.921 23.556-19.936a5.584 5.584 0 0 1 3.098-.86zm3.965 2.96a2.895 2.895 0 0 1 2.043.844 2.786 2.786 0 0 1 .879 2.121 2.869 2.869 0 0 1-.985 2.07l-24.25 21.106-2.617-2.617 22.887-22.68a2.895 2.895 0 0 1 2.043-.843zm2.994 6.698c-1.69 6.702-10.647 15.783-19.987 24.607l-3.777-3.773L86.3 49.088zM51.367 61.547l.274.27 3.513 3.513-9.63 2.06 5.843-5.843zm5.793 5.84.004.004 1.168 1.195a1.086 1.086 0 0 0 .018.084l.078.012.248.254.82.84-5.385.66 3.05-3.05zm3.867 4.076 3.578 3.576A126.992 126.992 0 0 1 38.75 91.695a1.44 1.44 0 0 0-.777 1.653l1.035 4.5a.31.31 0 0 1 0 .363.31.31 0 0 1-.414 0l-6.102-6.152L51.3 72.975l9.728-1.512zm-29.933 21.94.869.814 4.492 4.492-10.016-.648 4.655-4.659z"/>
```
Comment (Snailedlt) [inline icons/postman/postman-original.svg:4]: Change fill color to `#f37036`
```suggestion
    <path fill="#f37036" d="M91.95 23.31a11.047 11.047 0 0 0-7.759 3.17 10.988 10.988 0 0 0-2.39 11.641c-4.741-2.03-11.155 1.51-31.106 21.457a.932.932 0 0 0-.037.094 1.242 1.242 0 0 0-.119.062l-6.309 6.364a1.97 1.97 0 0 0-.363 2.324 2.012 2.012 0 0 0 1.707.984l.313-.203 8.424-1.797-4.03 4.067a.873.873 0 0 0-.054.166l-19.75 19.799a.798.798 0 0 0-.192.238l-5.086 5.09a1.967 1.967 0 0 0-.414 2.043 1.995 1.995 0 0 0 1.656 1.265l12.618.88a1.01 1.01 0 0 0 .52-.415.886.886 0 0 0 0-1.035l-.026-.025a2.243 2.243 0 0 0 .705-.58 2.237 2.237 0 0 0 .406-1.876l-.984-4.187a126.725 126.725 0 0 0 26.334-16.861 1.091 1.091 0 0 0 .248.103c.254-.019.492-.128.672-.308 13.55-12.83 21.515-21.622 21.515-28.602a8.03 8.03 0 0 0-.431-2.85 10.957 10.957 0 0 0 3.845.83l-.015.004a11.219 11.219 0 0 0 5.183-1.45.775.775 0 0 0 .004.001.835.835 0 0 0 .617-.055 9.398 9.398 0 0 0 2.07-1.652 10.873 10.873 0 0 0 3.258-7.758 10.873 10.873 0 0 0-3.257-7.758.93.93 0 0 0-.118-.091 11.045 11.045 0 0 0-7.656-3.078zm-.087 1.772a9.27 9.27 0 0 1 5.586 1.914l-8.068 8.117a.84.84 0 0 0-.076.098.83.83 0 0 0-.239.55.832.832 0 0 0 .313.65h.002l6.1 6.1a9.044 9.044 0 0 1-10.028-1.913c-2.586-2.6-3.336-6.504-1.953-9.891 1.383-3.39 4.68-5.605 8.363-5.625zm7.12 3.432a8.87 8.87 0 0 1 2.033 5.674 9.15 9.15 0 0 1-2.688 6.464 9.989 9.989 0 0 1-1.098.895L92.307 36.7l-.963-.963.265-.265 7.373-6.96zm-.366 4.193a.777.777 0 0 0-.55.031.731.731 0 0 0-.36.426.73.73 0 0 0 .05.559 2.226 2.226 0 0 1-.257 2.328.64.64 0 0 0-.195.488c.004.184.07.36.195.492a.58.58 0 0 0 .414 0 .68.68 0 0 0 .672-.207 3.573 3.573 0 0 0 .465-3.777v.004a.777.777 0 0 0-.434-.344zM79.34 39.43a5.584 5.584 0 0 1 3.31 1.226 4.756 4.756 0 0 0-2.681 1.34L57.162 64.701l-4.476-4.476c11.828-11.772 19.06-17.921 23.556-19.936a5.584 5.584 0 0 1 3.098-.86zm3.965 2.96a2.895 2.895 0 0 1 2.043.844 2.786 2.786 0 0 1 .879 2.121 2.869 2.869 0 0 1-.985 2.07l-24.25 21.106-2.617-2.617 22.887-22.68a2.895 2.895 0 0 1 2.043-.843zm2.994 6.698c-1.69 6.702-10.647 15.783-19.987 24.607l-3.777-3.773L86.3 49.088zM51.367 61.547l.274.27 3.513 3.513-9.63 2.06 5.843-5.843zm5.793 5.84.004.004 1.168 1.195a1.086 1.086 0 0 0 .018.084l.078.012.248.254.82.84-5.385.66 3.05-3.05zm3.867 4.076 3.578 3.576A126.992 126.992 0 0 1 38.75 91.695a1.44 1.44 0 0 0-.777 1.653l1.035 4.5a.31.31 0 0 1 0 .363.31.31 0 0 1-.414 0l-6.102-6.152L51.3 72.975l9.728-1.512zm-29.933 21.94.869.814 4.492 4.492-10.016-.648 4.655-4.659z"/>
```

File: icons/postman/postman-original-wordmark.svg [hunk 6]
```diff
@@ -0,0 +1,5 @@
...
...c0-.349-.098-.743.2-1.04l.01-.02c.336.146.388.493.536.79l2.77 6.136c.2.446.406.988.94.793.493-.153.345-.691.345-1.09v-9.156c0-.297-.102-.543-.399-.593zm-30.531.029a.722.722 0 0 0-.176.033c-.445.15-.344.691-.344 1.086v9.305c.047.246.149.445.446.496.285 0 .481-.2.531-.496.05-.297.05-.595.05-.89v-5.99c0-.186-.05-.484.2-.53.195-.051.195.246.297.394l1.332 2.922c.152.344.199.738.598.738.28 0 .382-.394.53-.691l1.634-3.512.007-.016c.395.399.294.793.344 1.188v5.691c0 .5-.156 1.192.535 1.192.645 0 .493-.696.493-1.14v-8.61c0-.448.136-.988-.395-1.14-.543-.098-.793.347-.988.792l-1.52 3.266c-.152.297-.2.89-.644.89-.4-.062-.446-.593-.594-.89l-1.524-3.364c-.17-.389-.42-.744-.812-.724zm-44.531.867c.476.006.933.12 1.328.516.746.789.742 1.78.496 2.818-.247 1.043-.789 1.389-2.227 1.389-1.386 0-1.386 0-1.386-1.586v-2.227c0-.348 0-.79.347-.79.471-.026.966-.126 1.442-.12zm11.824.068c1.138-.05 1.829.696 1.879 1.98v2.54c0 .891.05 1.742 0 2.574-.047 1.34-.742 2.028-1.88 2.028-1.105-.051-1.746-.747-1.796-2.028v-5.113c.066-1.29.712-1.98 1.797-1.98zm48.484 1.141c.297 0 .247.344.297.594l.942 3.512c.343 1.234.293 1.285-.793 1.285h-.496l.003.02c-1.585 0-1.636 0-1.14-1.798l.793-2.968c.05-.301.097-.645.394-.645z"/>
```
Comment (Snailedlt) [inline icons/postman/postman-original-wordmark.svg:4]: Change fill color to `#f37036`
```suggestion
    <path fill="#f37036" d="M28.723 53.139a2.96 2.96 0 0 0-1.864.838 2.93 2.93 0 0 0-.646 3.11 1.621 1.621 0 0 0-.676-.13c-1.256.025-3.188 1.42-7.63 5.863-.01.01-.013.02-.02.03a.33.33 0 0 0-.02.013l-1.687 1.7a.52.52 0 0 0-.094.62.535.535 0 0 0 .453.262l.098-.054 2.232-.479-1.072 1.084a.24.24 0 0 0-.04.11l-5.23 5.23a.316.316 0 0 0-.05.035l-1.38 1.383a.514.514 0 0 0-.113.547.518.518 0 0 0 .446.336l3.37.234v.004a.299.299 0 0 0 .137-.113.24.24 0 0 0 .012-.262.597.597 0 0 0 .32-.668l-.277-1.105a33.835 33.835 0 0 0 7.037-4.508.27.27 0 0 0 .037.008h.012a.296.296 0 0 0 .18-.082c3.622-3.426 5.75-5.777 5.75-7.641a2.124 2.124 0 0 0-.112-.742 2.93 2.93 0 0 0 1.006.203v-.016a2.98 2.98 0 0 0 1.37-.373.197.197 0 0 0 .013.002.227.227 0 0 0 .168-.012 2.51 2.51 0 0 0 .55-.441 2.896 2.896 0 0 0 .038-4.107.236.236 0 0 0-.012-.014.236.236 0 0 0-.025-.027 2.96 2.96 0 0 0-.022-.018.246.246 0 0 0-.068-.059 2.96 2.96 0 0 0-2.191-.761zm.18.466a2.48 2.48 0 0 1 1.492.512l-2.153 2.168a.209.209 0 0 0-.076.123v.004a.233.233 0 0 0-.006.045.217.217 0 0 0 0 .002v.002a.227.227 0 0 0 .006.027.217.217 0 0 0 .076.145l1.63 1.629a2.414 2.414 0 0 1-2.536-.38l-.008-.005c-.004-.004-.007-.01-.012-.014a.257.257 0 0 0-.007-.004 2.45 2.45 0 0 1-.118-.109 2.426 2.426 0 0 1-.52-2.64 2.424 2.424 0 0 1 2.231-1.505zm1.902.918c.35.425.543.96.543 1.516a2.43 2.43 0 0 1-.72 1.727 2.46 2.46 0 0 1-.278.23l-1.584-1.543 2.039-1.93zm-.159 1.106a.207.207 0 0 0-.084.02.247.247 0 0 0-.105.113c-.027.05-.012.105.012.152a.59.59 0 0 1-.07.621.184.184 0 0 0 0 .262.143.143 0 0 0 .113 0 .183.183 0 0 0 .18-.055.956.956 0 0 0 .124-1.008l.008-.004a.207.207 0 0 0-.178-.101zm-5.234 1.826a1.494 1.494 0 0 1 1.045.326 1.266 1.266 0 0 0-.73.356l-6.092 6.05L18.445 63c3.157-3.14 5.087-4.767 6.285-5.32a1.494 1.494 0 0 1 .682-.225zm1.084.768a.769.769 0 0 1 .668.218.733.733 0 0 1 .065.069 1.567 1.567 0 0 1 .12.21.733.733 0 0 1 .006.014.733.733 0 0 1 .043.27.77.77 0 0 1-.261.555L20.64 65.21l-.692-.691 6.121-6.079a.769.769 0 0 1 .426-.218zm.008.478a.458.458 0 0 0-.09.082l.09-.082zm.9 1.342c-.484 1.785-2.84 4.182-5.326 6.535l-1-1 6.326-5.535zm-9.318 3.3.027.026.985.985-2.575.552 1.563-1.562zm1.547 1.548.328.336a.302.302 0 0 0 .004.021l.017.002.3.305-1.534.232-.004-.008.889-.888zm1.012 1.103.98.97a34.022 34.022 0 0 1-6.91 4.45.387.387 0 0 0-.207.441l.277 1.204a.071.071 0 0 1 0 .093.083.083 0 0 1-.11 0l-1.628-1.656 5.045-5.117 2.553-.385zm-7.965 5.861.117.11 1.307 1.306-2.674-.173 1.25-1.243zm58.855-13.466a2.644 2.644 0 0 0-.547.05c-1.735.347-2.527 1.325-2.379 3.118.098 1.34.844 1.978 1.832 2.423.692.36 1.395.596 1.93 1.19a2.125 2.125 0 0 1 .445 2.226c-.3.843-.843 1.188-1.582 1.239-.793.039-1.437-.145-2.03-.79-.212-.25-.493-.45-.743-.154-.297.348-.047.694.148.944.65.788 1.484.986 2.524.986v-.012c.347-.1.89-.048 1.387-.396 1.089-.641 1.386-1.633 1.386-2.918 0-1.387-.593-2.336-1.683-2.871-.692-.344-1.434-.555-2.078-1.086-.496-.45-.595-1.04-.497-1.734.101-.692.606-1.04 1.141-1.137.645-.098 1.337-.098 1.98.25.298.144.64.343.79-.154.148-.43-.149-.727-.493-.829-.483-.184-.994-.34-1.53-.345zm36.342.02c-.527.033-.577.626-.717 1.04l-2.426 8.961c-.097.395-.246.891.25 1.086.493.137.543-.344.645-.691.593-1.798.593-1.797 2.273-1.797h.438c1.352 0 1.353 0 1.793 1.52.052.202.149.449.199.644.09.297.285.347.531.297.297-.051.348-.297.348-.645l-.008.028a5.705 5.705 0 0 0-.195-.891l-2.278-8.512c-.097-.442-.207-.989-.742-1.039a.81.81 0 0 0-.111-.002zm-48.373.032c-1.484 0-2.472.692-2.672 2.13-.297 2.22-.246 4.49 0 6.718.199 1.538 1.141 2.23 2.773 2.23 1.536 0 2.375-.757 2.622-2.277.15-1.087.052-2.124.101-3.207-.05-1.14.098-2.278-.152-3.465-.246-1.387-1.188-2.129-2.672-2.129zm20.441.004c-.245 0-.445.2-.445.496 0 .383.203.538.496.538s.642.046.938 0c.743-.102 1.089.197 1.039 1.134v7.969c0 .383 0 .828.496.828.446 0 .496-.433.496-.828v-8.115c0-.742.2-1.09.89-1.04.348.05.743 0 1.086 0 .301 0 .496-.187.496-.534 0-.293-.195-.448-.445-.448h-5.047zm-34.207.008c-.582 0-.828.246-.828.988v4.586l-.023.018v4.65c0 .395 0 .793.445.793s.492-.394.492-.742c0-.828.051-1.733 0-2.623-.047-.743.203-1.088.895-1.041.453.05.836 0 1.281 0 1.547-.098 2.438-1.12 2.488-2.918.051-2.625-.793-3.66-3.07-3.71h-1.68zm76.961.106c-.285 0-.481.152-.531.496-.051.246-.05.441-.05.691v5.8c0 .2.05.497-.196.599-.301.097-.3-.199-.399-.399-.148-.398-.347-.742-.496-1.137l-2.422-5.398c-.203-.395-.45-.744-.89-.594-.398.1-.348.547-.348.89v9.122c0 .394 0 .938.535.89.496 0 .395-.546.395-.89v-5.89c0-.349-.098-.743.2-1.04l.01-.02c.336.146.388.493.536.79l2.77 6.136c.2.446.406.988.94.793.493-.153.345-.691.345-1.09v-9.156c0-.297-.102-.543-.399-.593zm-30.531.029a.722.722 0 0 0-.176.033c-.445.15-.344.691-.344 1.086v9.305c.047.246.149.445.446.496.285 0 .481-.2.531-.496.05-.297.05-.595.05-.89v-5.99c0-.186-.05-.484.2-.53.195-.051.195.246.297.394l1.332 2.922c.152.344.199.738.598.738.28 0 .382-.394.53-.691l1.634-3.512.007-.016c.395.399.294.793.344 1.188v5.691c0 .5-.156 1.192.535 1.192.645 0 .493-.696.493-1.14v-8.61c0-.448.136-.988-.395-1.14-.543-.098-.793.347-.988.792l-1.52 3.266c-.152.297-.2.89-.644.89-.4-.062-.446-.593-.594-.89l-1.524-3.364c-.17-.389-.42-.744-.812-.724zm-44.531.867c.476.006.933.12 1.328.516.746.789.742 1.78.496 2.818-.247 1.043-.789 1.389-2.227 1.389-1.386 0-1.386 0-1.386-1.586v-2.227c0-.348 0-.79.347-.79.471-.026.966-.126 1.442-.12zm11.824.068c1.138-.05 1.829.696 1.879 1.98v2.54c0 .891.05 1.742 0 2.574-.047 1.34-.742 2.028-1.88 2.028-1.105-.051-1.746-.747-1.796-2.028v-5.113c.066-1.29.712-1.98 1.797-1.98zm48.484 1.141c.297 0 .247.344.297.594l.942 3.512c.343 1.234.293 1.285-.793 1.285h-.496l.003.02c-1.585 0-1.636 0-1.14-1.798l.793-2.968c.05-.301.097-.645.394-.645z"/>
```

**Review by Snailedlt — CHANGES_REQUESTED** [review]
The fill color is wrong according to their press/media page -> https://www.postman.com/company/press-media/

Needs to be changed in all .svg's and in devicon.json

image

**Comment by kilian-paquier** [comment]
Hello @Snailedlt, I've made the changes, you can close or not the threads 🙂

**Review by Snailedlt — DISMISSED** [review]
Nice!
Looks good to me now

**Review by Snailedlt — APPROVED** [review]
LGTM ✔️
Thanks for contributing! 💯

**Pull request merged**